content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -32,64 +27,33 @@ but not production. Like all alpha features, this guide is subject to change.
 
 ## Prepare the guide environment
 
-1. Set the environment variables `"${ISTIO_DIR}"`, `"${WORK_DIR}"`, your cluster
-    name, and the service namespace. Ensure `"${WORK_DIR}"` is prefixed with `"${HOME}"`
+1. Create a virtual machine
+1. Set the environment variables `"${VM_NAME}"`, `"${WORK_DIR}"`,`"${CLUSTER_NAME}"` , `"${VM_NAMESPACE}"`,
+，`"${SERVICE_ACCOUNT}"` your cluster name, and the service namespace. Ensure `"${WORK_DIR}"` is prefixed with `"${HOME}"`
```

Comment:
nit: does it really need to be prefixed with HOME?

Reply:
No it doesn't required, remove it